ddp-workspace/projects/ddp-dsm-ui/src/app/participant-page/participant-page.component.ts

```diff
@@ -110,7 +110,7 @@ export class ParticipantPageComponent implements OnInit, OnDestroy, AfterViewChe
   updatingParticipant = false;
   private taskType: string;
   private checkParticipantStatusInterval: any;
-  isEmailValid: boolean;
+  isEmailValid: boolean = true;
```

Comment:
I think it is not good idea to give value `true` directly, cause this variable is used to check whether email is valid or not, there is a special method called `validateEmailInput` in `participant-page.component.ts` which actually checks whether email is valid or not and updates value of `isEmailValid` accordingly, I think it is better to use that method in `ngOnInit`, you can read participant's email from `this.participant.data.profile`.

Reply:
Thank you for your feedback. I did as you suggested.